src/crimson/os/alienstore/alien_store.cc

```diff
@@ -270,24 +270,24 @@ AlienStore::readv(CollectionRef ch,
   });
 }
 
-AlienStore::get_attr_errorator::future<ceph::bufferptr>
+AlienStore::get_attr_errorator::future<ceph::bufferlist>
 AlienStore::get_attr(CollectionRef ch,
                      const ghobject_t& oid,
                      std::string_view name) const
 {
   logger().debug("{}", __func__);
-  return seastar::do_with(ceph::bufferptr{}, [=] (auto &value) {
+  return seastar::do_with(ceph::bufferlist{}, [=] (auto &value) {
     return tp->submit(ch->get_cid().hash_to_shard(tp->size()), [=, &value] {
       auto c =static_cast<AlienCollection*>(ch.get());
       return store->getattr(c->collection, oid,
 		            static_cast<std::string>(name).c_str(), value);
```

Comment:
int BlueStore::getattr(
  CollectionHandle &c_,
  const ghobject_t& oid,
  const char *name,
  bufferptr& value)
bluestore use bufferptr, if you transfer bufferlist,  how  does it handle bufferlist?

Reply:
there's ObjectStore::getattr which take a bufferlist parameter and invokes BlueStore's getattr